### PR TITLE
Fix problem with empty context.

### PR DESCRIPTION
**Pull request by mariuszstr**
I use django-graph-auth in my project.
But I problem:
-I had case with empty context, and django-graph-auth raises exception.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -61,7 +61,7 @@ Optional Settings
 .. code-block:: python
 
 	GRAPH_AUTH = {
-	    'USER_FIELDS': ('email', 'first_name', 'last_name', ), # Which username fields are available
+	    'USER_FIELDS': ('username', 'email', 'first_name', 'last_name', ), # Which user fields are available
```
Comment (morgante) [inline README.rst]: I don't think we should have this as a default. Instead, a note would be appropriate.

File: graph_auth/settings.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
     from django.utils import importlib  # Will be removed in Django 1.9
 
 DEFAULTS = {
-    'USER_FIELDS': ('email', 'first_name', 'last_name', ),
+    'USER_FIELDS': ('username', 'email', 'first_name', 'last_name', ),
```
Comment (morgante) [inline graph_auth/settings.py]: See above.

File: README.rst (hunk 1 above)
Comment (mariuszstr) [inline README.rst]: Why? I think username in user model is django default, and from my perspective username shoud be default.

Comment (morgante) [inline README.rst]: @mariuszstr What happens if it's not included? In general I think we should default to including/requiring the minimum fields necessary for the functionality to work and let other users add more as desired.

Comment (mariuszstr) [inline README.rst]: Ok, i've removed username, but still I think that username shoud be in default config instead of email :-)

**Comment by mariuszstr** [comment]
@morgante What about this PR? I removed username as You requested.

**Pull request merged**